lib/Common/CommonDefines.h

```diff
@@ -140,7 +140,11 @@
 #define ENABLE_JS_ETW                               // ETW support
 #else
 #define SYSINFO_IMAGE_BASE_AVAILABLE 0
+#ifndef ENABLE_VALGRIND
 #define ENABLE_CONCURRENT_GC 1
+#else
+#define ENABLE_CONCURRENT_GC 0
```

Comment:
@obastemur Yes, I was saying "no matching define to 0" elsewhere.
@leirocks Good point